packages/account-v2/src/pages/LoginHistory/LoginHistoryTable.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Table } from '@deriv-com/ui';
+
+export const LoginHistory = () => {
+    // const { data, header } = useLoginHistoryTableData();
+
+    return (
+        <div className='flex flex-col '>
+            <Table
+                columns={[
+                    {
+                        header: 'Date and Time ',
+                    },
+                    {
+                        header: 'Action',
+                    },
+                    {
+                        header: 'Browser',
+                    },
+                    {
+                        header: 'IP Address',
+                    },
+                    {
+                        header: 'Status',
+                    },
+                ]}
+                data={[
+                    {
+                        action: 'Login',
+                        browser: 'Chrome v122.0.0.0',
+                        datetime: '2024-03-07 12:04:42 GMT',
+                        IpAddress: '942',
+                        status: 'Successful',
+                    },
+                ]}
+                isFetching={true}
+                renderHeader={header => <span>{header}</span>}
+                rowRender={data => (
+                    <div className='grid grid-flow-col'>
+                        <div>{data.action}</div>
+                        <div>{data.browser} </div>
+                        <div>{data.datetime}</div>
+                        <div>{data.IpAddress} </div>
+                        <div>{data.status} </div>
+                    </div>
+                )}
```

Comment:
rowRender function is defined inside a component's render method, it will be recreated every time the component renders. Best to move it out and map over it. Something like this maybe
```suggestion
const rowRender = (data: { action: string; browser: string; datetime: string; IpAddress: string; status: string; }) => (
    <div className='grid grid-flow-col'>
        {Object.keys(data).map(key => (
            <div key={key}>{data[key as keyof typeof data]}</div>
        ))}
    </div>
);
```

Reply:
As suggested by @thisyahlen-deriv 
However if this doesn't preserve the order of cells, you can create an array like
```
const cellOrder = ['action', 'browser', 'datetime', 'IpAddress', 'state']


const rowRender = (data) => (
	    <div className='grid grid-flow-col'>
	        {cellOrder.map(key => (
	            <div key={key}>{data[key as keyof typeof data]}</div>
	        ))}
	    </div>
	);
```

`as keyof typeof data` can also be fixed/removed based on the data